### ISSUE TITLE
Dropdown programmatic open/close no longer works

### ISSUE THREAD

**Issue by dorisip** [themesberg/flowbite-svelte#256]
**Describe the bug**
In the Dropdown component, the programatic closing functionality no longer works

**To Reproduce**
Can reproduce in the example from https://flowbite-svelte.com/dropdowns#Programatic_open/close 

**Expected behavior**
It should close the dropdown

**Additional context**
I think this is broken as of 0.26.0

**Env Info**
```
System:
    OS: macOS 12.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 183.03 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 17.3.0 - ~/.nvm/versions/node/v17.3.0/bin/node
    npm: 8.3.0 - ~/.nvm/versions/node/v17.3.0/bin/npm
  Browsers:
    Chrome: 104.0.5112.101
    Firefox: 104.0.1
    Safari: 15.6
  npmPackages:
    svelte: ^3.49.0 => 3.49.0
```

**Comment by shinokada** [comment]
Can you post the output of `npx envinfo --system --npmPackages svelte,rollup,webpack --binaries --browsers`?

**Comment by dorisip** [comment]
> Can you post the output of `npx envinfo --system --npmPackages svelte,rollup,webpack --binaries --browsers`?

I pasted it in the description

**Comment by shinokada** [comment]
Thanks. Which version of Flowbite-Svelte are you using? 
If the version is not v0.26.2, can you update it and let me know if it has the same problem?

```
npm i -D flowbite-svelte@latest
```

**Comment by dorisip** [comment]
> Thanks. Which version of Flowbite-Svelte are you using? If the version is not v0.26.2, can you update it and let me know if it has the same problem?
> 
> ```
> npm i -D flowbite-svelte@latest
> ```

I'm on 0.26.2

**Comment by shinokada** [comment]
Can you test if other dropdowns and tooltips work?

**Comment by dorisip** [comment]
> Can you test if other dropdowns and tooltips work?

The dropdowns/tooltips do open on click, the problem is I cannot close it programmatically like in the example shown [here](https://flowbite-svelte.com/dropdowns#Programatic_open/close)

I think the issue is in this line: https://github.com/themesberg/flowbite-svelte/blob/main/src/lib/utils/Popper.svelte#L19 (it should be `export let open = false;` instead)

**Comment by shinokada** [comment]
@jjagielka

**Comment by shinokada** [comment]
@dorisip We have updated the component. Can you check and let us know.
I close the issue for now.
Thank you.

**Comment by dorisip** [comment]
@shinokada it looks like it's still broken so I can't test it properly

I'm on the latest version, 0.26.18, and the `on:click` is not being triggered on the `DropdownItem` component

**Comment by xyassini** [comment]
I can confirm that the on:click method currently doesn't work on DropdownItems

**Comment by xyassini** [comment]
@dorisip should be fixed as soon as #299 is released :)

**Comment by shinokada** [comment]
I close the issue for now. Please let me know if this release didn't fix the problem.
